docs/add-new-shops.md

```diff
@@ -24,4 +24,9 @@ If you want to add another to-be-scraped-shop, unfortunately, there are some fil
    - import the implemented function: `extract_<shop-name>` from the created file: `extract.extract.extractors.<shop-name>.py`
    - add the function for to the mapping: `EXTRACTOR_FOR_TABLE_NAME`
 6. [`workers.workers.__init__.py`](../workers/workers/__init__.py):
-   - add connection for `<shop-name>` to the mapping `CONNECTION_FOR_TABLE`
\ No newline at end of file
+   - add connection for `<shop-name>` to the mapping `CONNECTION_FOR_TABLE`
+7. [`start-job.scripts.<shop-name>.py`](../start-job/scripts/):
+   - create a new file `<shop-name>.py`. (Check out others for examples)
+   - TODO: could you finalize this?
+8. [`start-job.scripts.main.py`](../start-job/scripts/main.py):
+   - TODO: could you finalize this?
```

Comment:
I recognized that there is some missing documentation here, could you fix this?

Step 8. was missing in your code, that's why it bubbled to my attention.

Reply:
i wrote start_urls is a single url. is this correct? if not how would multiple urls be formatted.
also what is the union in `def get_settings() -> List[Union[Dict[str, str], Dict[str, Tuple[str, Dict[str, str]]]]]` about?

Reply:
That's right. The `Union` was used because of the `meta_data` entry. For other scrapers its shape could be very different. Fixed this by using `List[dict]` as type.

Reply:
ah good. i was worried that i missed something.